Tiltfile

```diff
@@ -5,7 +5,13 @@ update_settings(k8s_upsert_timeout_secs=300)
 
 # Setup a development registry so we can push images to it
 # and use them to test the scanner.
-k8s_yaml("./hack/registry.yaml")
+# To setup a private registry, set `use_private_registry: true`
+# on the configuration file.
+use_private_registry = settings.get("use_private_registry", False)
```

Comment:
could you update the tiltfile settings example file, adding this setting commented out?